samples/react-sample-app/src/useAuthProvider.js

```diff
@@ -0,0 +1,160 @@
+import { useState, useEffect } from "react";
+import {
+    msalApp,
+    requiresInteraction,
+    fetchMsGraph,
+    isIE,
+    GRAPH_ENDPOINTS,
+    GRAPH_SCOPES,
+    GRAPH_REQUESTS
+} from "./auth-utils";
+
+// If you support IE, our recommendation is that you sign-in using Redirect APIs
+const useRedirectFlow = isIE();
+// const useRedirectFlow = true;
+
+export default function useAuthProvider() {
+    const [account, setAccount] = useState(null);
+    const [error, setError] = useState(null);
+    const [emailMessages, setEmailMessages] = useState(null);
+    const [graphProfile, setGraphProfile] = useState(null);
+
+    useEffect(() => {
+        async function doAuth() {
+            msalApp.handleRedirectCallback(error => {
+                if (error) {
+                    const errorMessage = error.errorMessage
```

Comment:
I think `error.errorMessage || "Unable to acquire access token."` would be valid here instead of the ternary.